lib/url-pattern/url-match-pattern-list.js

```diff
@@ -0,0 +1,95 @@
+var _ = require('../util').lodash,
```

Comment:
Is this class really necessary? I'm not sure why we need it at all. It has only one function, `test()`, which IMO can be implemented as a static method on `UrlMatchPattern`. Is there any future use of this? 

@shamasis Thoughts?